src/lib/index.js

```diff
@@ -344,6 +344,17 @@ lib.mergeArray = function(traceAttr, cd, cdAttr) {
     }
 };
 
+lib.getTargetArray = function(trace, target) {
+    if(typeof target === 'string' && target) {
+        var array = lib.nestedProperty(trace, target).get();
+
+        return Array.isArray(array) ? array : [];
```

Comment:
done in 2dd1333a0706c91628dc596d147e6f88d18b5183 , 9c70ee963d5af87c81e84af01cda3da81d45dbab and  4f9195785367c253fd27c4b5471b33a45e3e19b6